### DISCUSSION TITLE
从recV3-recv4速度变慢

### DISCUSSION THREAD

**Discussion by yby1213** [PaddlePaddle/PaddleOCR#12846] [Q&A]
我想请问一下我使用PP-OCRv4-server-rec转成onnx格式的文件，在djl上运行，为啥耗时上比之前我使用的V3版本（ch_PP-OCRv3_rec）高了7-8秒，请问是啥原因引起的

**Comment by hezhenname** [comment]
正常吧，PP-OCRv4-server-rec模型是高精度模型，100+M

**Comment by polarisunny** [comment]
请问DJL是怎么运行PPOCR的三个模型的？一直没弄明白，谢谢！
